server/src/main/java/org/elasticsearch/common/util/Maps.java

```diff
@@ -292,7 +292,7 @@ public static <K, V> Map<K, V> newHashMapWithExpectedSize(int expectedSize) {
      * @return a new pre-sized {@link HashMap}
      */
     public static <K, V> Map<K, V> newConcurrentHashMapWithExpectedSize(int expectedSize) {
-        return new ConcurrentHashMap<>(capacity(expectedSize));
+        return new ConcurrentHashMap<>(expectedSize);
```

Comment:
Similar to the above note, this utility method doesn't seem to be doing anything anymore, so perhaps it should be removed.

Reply:
(here and above comment) Yeap, newMapWithExpectedSize, newHashMapWithExpectedSize, newConcurrentHashMapWithExpectedSize, newLinkedHashMapWithExpectedSize can all be inlined.